src/storage.rs

```diff
@@ -92,19 +93,39 @@ pub struct Eip7702Code {
     version: u8,
 }
 
+/// EVM Object Format (EOF) code.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct EofCode {
+    /// EOF header.
+    pub header: EofHeader,
+    /// EOF body.
+    pub body: EofBody,
```

Comment:
I am not very clear shall we use any filed of `EofHeader/EofBody` in pevm， as my understanding, we just save the contract code in mv_memory. If we don't use any fileds of this struct, then can we save the raw code directly?

Reply:
That's a sharp insight! This is a tradeoff as making our structs closer to `revm` would make conversions faster -- also why we don't store raw bytes and re-analyze on conversion for legacy bytecodes. If decoding raw bytes is fast enough for EOF, we can store raw bytes here, and optimize later if profiling shows that it's a non-trivial bottleneck.

Reply:
Since EOF bytecodes are not yet popular, I'm happy to store the raw bytes here and optimise the conversion in the future, once EOF gains more traction 🙏.

Reply:
@hai-rise, Thanks for your quick reply, will refactor soon!